regexes/device/mobiles.yml

```diff
@@ -2010,6 +2047,14 @@ Dicam:
   device: 'feature phone'
   model: '$1'
 
+# Digicel
+Digicel:
+  regex: 'DIGICEL'
+  device: 'smartphone'
+  models:
+    - regex: '(DL810)'
+      model: '$1'
```

Comment:
Can be simplified to 
```
    - regex: 'DL810'
      model: 'DL810'
```
Also it might be useful to have a global device matching, eg
```
    - regex: 'DIGICEL ([^/;]+) Build'
      model: '$1'
```